test/AssemblyContext.cpp

```diff
@@ -82,7 +83,8 @@ INSTANTIATE_TEST_CASE_P(
       // A very long string, encoded after an initial word.
       // SPIR-V limits strings to 65535 characters.
       {std::string(65535, 'a'), {1}},
-    }));
+    }),
+    &DefaultParameterPrinter<EncodeStringCase>);
```

Comment:
This is onerous (we must add junk to all INSTANTIATE calls now and forever) and surprising (we can't use INSTANTIATE the way Googletest doc suggest). There has to be a better way. :pray: